content/en/platform/corda/5.0-dev-preview-1/cordapps/integration-tests.md

```diff
@@ -230,7 +230,7 @@ class IOUContract : Contract {
 
 ## Deploy your CorDapp
 
-Now, [deploy your sample CorDapp to a local Corda 5 development network](deploy-cordapps.md).
```

Comment:
@ivanterziev-r3 @michelemarsden I'm concerned with how we're fixing links with this project more generally. For example, this link works on the live site. You can find it [here](https://docs.r3.com/en/platform/corda/5.0-dev-preview-1/cordapps/integration-tests.html#deploy-your-cordapp).

Why are we overcomplicating?

Reply:
The links here work indeed. Let's discuss outside of this PR.